### PR TITLE
Clarify custom domain docs

### PR DESCRIPTION
**Pull request by liftedkilt**

### CODE REVIEW

File: intro/custom-domains.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ After configuring the custom domains on the Project Settings, you must point you
 
 ## Configuring DNS for wildcard domain
 
-Since you can have multiple services inside a project and WeDeploy automatically creates a subdomain for each service. Wildcard domains allow you to map any of all generated subdomains from the services with a single record.
+WeDeploy automatically creates a subdomain for each service. Since you typically will have multiple services inside a project, you will either need multiple records, or a single Wildcard record. A wildcard record allows you to map all subdomains to your wedeploy project with a single record.
```
Comment (zenorocha) [inline intro/custom-domains.md]: `wedeploy` > `WeDeploy`

Comment (zenorocha) [inline intro/custom-domains.md]: `single Wildcard record` > `single wildcard record`

**Review by zenorocha — CHANGES_REQUESTED** [review]

**Comment by zenorocha** [comment]
Thanks for the PR @liftedkilt!

**Comment by liftedkilt** [comment]
@zenorocha fixed. Thanks for catching that.

**Comment by zenorocha** [comment]
Thanks!!

**Comment by eduardolundgren** [comment]
@liftedkilt thanks!

**Pull request merged**